### PR TITLE
Fix decoding unicode values in the config object passed to the browser

### LINKED ISSUE #5435: Cypress.env() return value is garbled when multibyte string in 3.5.0
### Current behavior:

Return value of `Cypress.env()` is garbled when `cypress.env.json` includes multibyte string.

### Desired behavior:

Return value of `Cypress.env()` is not garbled.

On cypress 3.4.1, I got desired behavior.

### Steps to reproduce: (app code and test code)

- Create following files
```js
// cypress/integration/garbled_test.js
const myenv = Cypress.env()
describe('garbled test', function() {
  it('garbled test', function() {
    expect(myenv.alphabet).to.equal('aiueo')
    expect(myenv.multibyte).to.equal('あいうえお')
  })
})
```

cypress.env.json

```json
{
  "alphabet": "aiueo",
  "multibyte": "あいうえお"
}
```

`あいうえお` is Japanese characters.

- Execute `$ npx cypress open` and select `garbled_test.js`

image

### Versions

- Cypress 3.5.0
- macOS Mojave

### PR DESCRIPTION
**Pull request by bahmutov**
- Closes #5435 
- related to https://github.com/cypress-io/cypress/pull/5155
- failing demo in 3.5.0 passes with this fix https://github.com/cypress-io/cypress-example-recipes/pull/353

### User-facing changes

Environment variables with unicode values are correctly passed to the runner again. In 3.5.0 we have accidentally broken them by encoding resolved Cypress config object incorrectly. This became a problem for anyone using Unicode characters in their environment variables in the configuration file like

```json
{
  "env": {
     "greeting": "привет"
  }
}
```

### PR Tasks

- [x] Have tests been added/updated?

### CODE REVIEW

File: packages/runner/src/lib/util.js [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+  // 'eyJzdGF0ZSI6IvCfmYIifQ=='
+  // "window.atob" does NOT work
+  // atob('eyJzdGF0ZSI6IvCfmYIifQ==')
+  // "{"state":"ð"}"
+  // but this function works
+  b64DecodeUnicode('eyJzdGF0ZSI6IvCfmYIifQ==')
+  '{"state":"🙂"}'
+  ```
+ */
+  b64DecodeUnicode (str) {
```
Comment (jennifer-shehane) [inline packages/runner/src/lib/util.js]: Add comment linking to original issue: 

```
// https://github.com/cypress-io/cypress/issues/5435
```

File: packages/runner/src/lib/util.spec.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import util from './util'
+
+describe('util', () => {
+  context('b64DecodeUnicode', () => {
+    it('decodes unicode string correctly', () => {
+      const s = '🙂 привет 🌎'
```
Comment (jennifer-shehane) [inline packages/runner/src/lib/util.spec.js]: Add comment linking to original issue: 

```
// https://github.com/cypress-io/cypress/issues/5435
```

**Review by jennifer-shehane — COMMENTED** [review]
I confirmed that this fixes the original issues bug with the `あいうえお` characters.

I didn't review the code specifically.

**Review by flotwig — APPROVED** [review]
Wow, didn't know that atob didn't work with Unicode. Nice fix, LGTM

**Pull request merged**